src/vm-repair/setup.py

```diff
@@ -8,10 +8,8 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-
 VERSION = "1.1.0"
```

Comment:
Please upgrade the extension version in the `setup.py` to trigger the release of new extension version